model/driver/src/icon4py/model/driver/icon4py_configuration.py

```diff
@@ -21,6 +32,38 @@
 n_substeps_reduced = 2
 
 
+class DriverBackends(str, enum.Enum):
+    GTFN_CPU = "gtfn_cpu"
+    GTFN_CPU_CACHED = "gtfn_cpu_cached"
+    GTFN_GPU = "gtfn_gpu"
+    GTFN_GPU_CACHED = "gtfn_gpu_cached"
+
+
+backend_map: Final[dict] = {
+    DriverBackends.GTFN_CPU: run_gtfn,
+    DriverBackends.GTFN_CPU_CACHED: run_gtfn_cached,
+    DriverBackends.GTFN_GPU: run_gtfn_gpu,
+    DriverBackends.GTFN_GPU_CACHED: run_gtfn_gpu_cached,
+}
+
+
```

Comment:
Something like that I meant above. But I think it is better to have it in `common` and then use the properties of the `gtx.Backend`s `DeviceType` to determine which import to use (see above), because then we can reuse it in all model packages.

Reply:
Thanks. I have removed this function and used the one as suggested in your comment above.